pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -813,6 +814,204 @@ func TestUserSignupWithInvalidNameNotOK(t *testing.T) {
 	)
 }
 
+func TestUserSignupDeactivatedWithNoMUR(t *testing.T) {
+	userSignup := &v1alpha1.UserSignup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      uuid.NewV4().String(),
+			Namespace: operatorNamespace,
+			UID:       types.UID(uuid.NewV4().String()),
```

Comment:
I believe you don't need to set the `UID`

Reply:
This was an artifact of copying the previous tests, I've removed it from this PR.